oxidation/libparsec/crates/types/Cargo.toml

```diff
@@ -6,15 +6,14 @@ license = " BUSL-1.1"
 autotests = false
 
 [[test]]
-name = "integration"
+name = "types_integration"
 path = "tests/mod.rs"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [features]
 # TODO: this feature is never referenced, is it useful ?
 js = ["uuid/js"]
 test-mock-time = []
+types-fixtures = ["dep:hex-literal", "dep:rstest"]
```

Comment:
rename to `test-fixtures`
the `test-` prefix should be used for everything that basically should not be enabled when in production
the `types-` part is not really informative given we are already in the `types` crate

Reply:
done